packages/components/src/FeatureSwitch/FeatureSwitch.tsx

```diff
@@ -109,7 +109,6 @@ export function FeatureSwitch({
                 transition={{
                   delay: 0.2,
                   type: "spring",
-                  duration: 0.2,
```

Comment:
We are able to remove these durations as they were not used.

When we look at that [docs](https://www.framer.com/api/motion/types/#spring) for the "spring" type, it does not accept a duration option.